src/models/darp10/repairs.md

```diff
@@ -0,0 +1,287 @@
+    ![Speaker wires and screws (darp10, 16")](./img/speakers-16.webp)
+    - For the 14" model (darp10-b), lift the speakers up and off of the plastic posts.
+    ![Speaker wires and posts (darp10-b, 14")](./img/speakers-14.webp)
+5. Slide the new speakers onto the plastic posts and connect them to the motherboard.
+    - The left speaker's wire can be run through the channel in the chassis before or after plugging it in.
+
+## Replacing the keyboard:
+
+The keyboard can be replaced if its switches or electronics have been damaged.
+
+**Part numbers:**
+- 16" (darp10) keyboard: `6-23-RV56T-011`
+- 14" (darp10-b) keyboard: `6-23-RL245-010`
+- Keyboards shipped by System76 include a custom-printed Super key.
+
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 5 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the keyboard:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the keyboard screws (highlighted yellow below).
+    - On the 16" model (darp10), there is one keyboard screw.
+    ![Keyboard screws (16")](./img/keyboard-screw-16.webp)
+    - On the 14" model (darp10-b), there are two keyboard screws.
+    ![Keyboard screws (14")](./img/keyboard-screws-14.webp)
+3. For the 16" model (darp10) only, you can optionally [remove the bottom panel](), [remove the heatsink](), and slowly pull the tabs highlighted below to remove the two keyboard adhesive strips.
+    - If you skip this step, pull the keyboard out slowly to avoid tearing its material on the adhesive.
+    ![Keyboard adhesive (16")](./img/keyboard-adhesive-16.webp)
+4. Open the lid slightly and place the machine on its side.
+5. Push the screwdriver into the keyboard push point (highlighted cyan above) until the keyboard pops out.
+
+![Keyboard push point](./img/keyboard-push-point.webp)
+
+6. Set the machine back down and raise the keyboard away from the chassis. The larger ribbon cable is for the keyboard, while the smaller ribbon cable is for the keyboard backlight.
+    - On the 16" model (darp10), the keyboard backlight cable is to the right of the main cable.
+    ![Keyboard ribbons (16")](./img/keyboard-ribbons-16.webp)
+    - On the 14" model (darp10-b), the keyboard backlight cable is to the left of the main cable.
+    ![Keyboard ribbons (14")](./img/keyboard-ribbons-14.webp)
+7. Flip the black latches upwards to free the ribbon cables.
+8. Pull the ribbon cables out of the connectors.
+9. Remove the keyboard and replace it with the new one.
+10. Carefully slide both ribbon cables into their connectors.
+11. For the 16" model (darp10) only, optionally replace the keyboard adhesive strips (if they were removed earlier).
+    - The keyboard can be installed without adhesive strips if they're no longer usable.
+12. Flip the black latches back into place to secure the ribbon cables.
```

Comment:
Should steps 11 and 12 be swapped? I feel like cable latches should follow immediately after plugging the cables into the sockets.

Reply:
Yes.